### PR TITLE
(feature) implement loading exchange and inserting exchange upon conf…

### PR DESCRIPTION
**Pull request by mark-antony1**
- create exchange on config creation
- load config via sqlite db

### CODE REVIEW

File: merkato/utils/database_utils.py [hunk 1]
```diff
@@ -73,21 +73,21 @@ def create_exchanges_table():
 		finally:
```
Comment (needmoney90) [inline merkato/utils/database_utils.py]: The indentation in this file is different from the rest of the project

File: merkato/merkato_config.py [hunk 2]
```diff
@@ -3,43 +3,35 @@
...
+    exchange_name = input("what is the exchange name? ")
+    return get_exchange(exchange_name)
+    # TODO: Error handling and config validation
+
+def insert_config_into_exchanges(config)
+    limit_only = config["limit_only"]
+    public_key = config["public_key"]
+    private_key = config["private_key"]
+    exchange = config["exchange"]
+    if no_merkatos_table_exists():
```
Comment (needmoney90) [inline merkato/merkato_config.py]: Why is this checking whether merkatos exists, then creating the exchange table? Shouldnt it be creating the merkatos table? (Or checking if the exchanges table exists)

Comment (needmoney90) [reply merkato/merkato_config.py]: I'm assuming the if no merkatos table check should instead be a check against the exchanges table, makes sense in context

**Review by needmoney90 — COMMENTED** [review]
Those two comments are my only concerns here, everything else looks great

File: merkato/merkato_config.py (hunk 2 above)
Comment (mark-antony1) [reply merkato/merkato_config.py]: yup typo

File: merkato/utils/database_utils.py (hunk 1 above)
Comment (mark-antony1) [reply merkato/utils/database_utils.py]: done

**Review by needmoney90 — APPROVED** [review]

**Comment by needmoney90** [comment]
Fixes #69

**Pull request merged**